### DISCUSSION TITLE
What version of Elasticsearch does this support?

### DISCUSSION THREAD

**Discussion by zanematthew** [Jeroen-G/Explorer#83] [General]
The demo app [here](https://github.com/Jeroen-G/explorer-demo/blob/main/docker-compose.yml#L41) is using elasticsearch `7.9.3`, but it isn't clear if `8.0.x` is supported 🤞 it is.

**Comment by Jeroen-G** [comment]
I am not working with ES 8 yet, but it is nice to know that it works :)

**Reply by moeen-basra** [reply]
I've been working with latest 8.3.2 and so far found no issues.
